components/style/values/specified/length.rs

```diff
@@ -576,7 +576,6 @@ impl CalcLengthOrPercentage {
         let mut ch = None;
         let mut rem = None;
         let mut percentage = None;
-        let mut number = None;
```

Comment:
have you tested what other browsers do? I remember this bit being weird and inconsistent between browsers. In particular, Firefox doesn't get calc right.